### ISSUE TITLE
bgp redundant address family attribute 

### ISSUE THREAD

**Issue by segevere** [IETF-OPSAWG-WG/l3nm#81]
inet:ip-address is IPv6 or IPv4 so there is no need to explicit the family address-family attribute.

**Comment by segevere** [comment]
the same goes to vpn-node , there is no reason to indicate the family since inet::ip-address include both.

**Comment by sbarguil** [comment]
I agree if we have an IPv4 configured in the neighbor or in the Router-ID the address can be automatically deduced. 

For example, if the neighbor is 10.0.0.2 the address-family ipv4 unicast can be expanded by the controller. 

```
R1(config-router)#address-family ipv4 unicast 
R1(config-router-af)#neighbor 10.0.0.2 activate
```

**Comment by boucadair** [comment]
> inet:ip-address is IPv6 or IPv4 so there is no need to explicit the family address-family >attribute.

AF may be useful to indicate whether the same session is used for both IPv4 and IPv6.

That is said, the current model does allow to cover the case where two sessions will be enabled; one for each AF.

**Comment by oscargdd** [comment]
Closed with sha  80914de69282f3f0024c483c27a718d0d3fee0be
